src/main/java/com/yellow/ordermanageryellow/jobs/OrdersNotificationJob.java

```diff
@@ -0,0 +1,31 @@
+package com.yellow.ordermanageryellow.jobs;
+
+import com.yellow.ordermanageryellow.model.Orders;
+import com.yellow.ordermanageryellow.service.OrdersService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.transaction.annotation.Transactional;
+import com.yellow.ordermanageryellow.Exception.*;
+import java.util.List;
+
+public class OrdersNotificationJob {
+    @Autowired
+    private OrdersService orderService;
+    @Scheduled(cron = "0 0,30 * * * ?")
+    @Transactional
```

Comment:
No need @Trasanctional

Reply:
Can you help me?-
I am trying to think where I have to change the flag, I don't think anyone did at all that the status would change,
When is this supposed to happen? I have to write it from the begining
Thanks